db/migrate/20221107195559_drop_iso_datastore.rb

```diff
@@ -0,0 +1,12 @@
+class DropIsoDatastore < ActiveRecord::Migration[6.1]
+  def up
+    drop_table :iso_datastores
+  end
+
+  def down
+    create_table :iso_datastores, force: :cascade do |t|
+      t.bigint :ems_id
+      t.datetime :last_refresh_on
```

Comment:
Was this table really just 2 columns (and id)? Seems almost pointless 😮

Reply:
Yes there could only be one per EMS so there wasn't even an ems_ref or name "needed" even though it would be helpful